src/client.ts

```diff
@@ -60,9 +59,38 @@ import {
 import {Transaction as LegacyTransaction} from "./legacy_transaction";
 import {Transaction, START_TRANSACTION_IMPL} from "./transaction";
 
-const PROTO_VER_MAJOR = 0;
-const PROTO_VER_MINOR_MIN = 9;
-const PROTO_VER_MINOR = 11;
+class ProtocolVersion {
```

Comment:
The protocol version is also used in 'src/codecs/registry', I think that may also need updating to use this class.

Reply:
changed